api/c/indigo/src/indigo_layout.cpp

```diff
@@ -48,48 +49,56 @@ CEXPORT int indigoLayout(int object)
                     f.unhide(submol.vertices[i]);
                 }
             }
-            MoleculeLayout ml(*mol, self.smart_layout);
-
-            if (obj.type == IndigoObject::SUBMOLECULE)
+            if (mol->isSequence())
             {
-                ml.filter = &f;
+                SequenceLayout sl(*mol);
+                sl.make();
             }
-
-            ml.max_iterations = self.layout_max_iterations;
-            ml.bond_length = MoleculeLayout::DEFAULT_BOND_LENGTH;
-            ml.layout_orientation = (layout_orientation_value)self.layout_orientation;
-            if (self.layout_preserve_existing || mol->hasAtropoStereoBonds())
```

Comment:
Looks like you accidentally delete option "layout_preserve_existing"

Reply:
fixed